### ISSUE TITLE
W* -  Training images moved from vo_agam_release to vo_agam_release_master_us_central1

### ISSUE THREAD

**Issue by jonbrenas** [anopheles-genomic-surveillance/anopheles-genomic-surveillance.github.io#233]
The links for all the images that are stored on gcs are broken due all of them having been moved from vo_agam_release to vo_agam_release_master_us_central1.

**Comment by jonbrenas** [comment]
@alimanfoo pointed out that the same is true for the reference genomes used by IGV. 

The preferred solution seems to be to create a permanent (and authentication-less) folder on S3 containing these resources before the notebooks are updated to point at the correct locations.

**Comment by ahernank** [comment]
Thanks @jonbrenas! For the URLs, would it be okay to collect the ones that are broken for the workshops you are checking & paste in here? I can then send you new URLs for the images.

**Comment by ahernank** [comment]
Btw, this issue is due to the fact that authentication is now required for vo_agam_release. The other bucket is not relevant here. :)

**Comment by jonbrenas** [comment]
> Btw, this issue is due to the fact that authentication is now required for vo_agam_release. The other bucket is not relevant here. :)

Yes, but all images have also been copied into the other bucket. They might not be needed there.

> Thanks @jonbrenas! For the URLs, would it be okay to collect the ones that are broken for the workshops you are checking & paste in here? I can then send you new URLs for the images.

I would say, generally, all the pictures in https://storage.googleapis.com/vo_agam_release/reference/training-images/ but, more specifically:

W1M2:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M2-1.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M2-2.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M2-3.png

W1M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M4-1.PNG
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M4-2.PNG

W2M1:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W2M1-1.PNG

W2M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W2M4-1new.PNG
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W2M4-2new.PNG

W3M3:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/gcp-locations.png

W3M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/behavioural-isolation.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/geographic-isolation.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/heightweight2.gif
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/PCA1-smaller-smaller.gif

W4M3:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-4/tree.png

W4M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-4/collection-sites-tz.png

W5M1 (despite the path for the first one):

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-4/mosquito-genotype-array.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-5/itn_use.png

W5M4 (thanks @KellyLBennett):

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-5/heterozygosity.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-5/bottleneck.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-5/roh_scatter.svg

W6M3:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/figure1_resized-768x489.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/Screenshot%20from%202022-09-01%2010-37-53.png

W7M1 (despite the paths):

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/429502_1_En_3_Fig4_HTML(1).jpg
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/Screenshot%20from%202022-10-17%2009-41-44.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/Screenshot%20from%202022-10-24%2019-22-07.png

W7M2:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/introgression-cartoon.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/spatial-gene-flow-cartoon.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/adaptive-gene-flow.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/haplotype-frequencies.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/detecting-sweeps.sketchpad.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/detecting-shared-sweeps.sketchpad.png

**Comment by ahernank** [comment]
Yup, that's true. You are right that we should have a piece of work cleaning-up what we have on those buckets, when we finish moving the resources to S3.

Thanks for listing the URLs @jonbrenas - it is quite helpful to see them this way to try to clean-up the names a bit. Updated URLs below, please let me know if you find any issues with these: 

W1M2:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M2-1.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-1/w1m2-1.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M2-2.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-1/w1m2-2.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M2-3.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-1/w1m2-3.png

W1M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M4-1.PNG-> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-1/w1m4-1.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W1M4-2.PNG-> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-1/w1m4-2.png

W2M1: 

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W2M1-1.PNG (I think link here should be https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-2/W2M1-1.PNG) -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-2/w2m1-1.png

W2M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W2M4-1new.PNG (https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-2/W2M4-1new.PNG) -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-2/w2m4-1.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-1/W2M4-2new.PNG (https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-2/W2M4-2new.PNG) -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-2/w2m4-2.png

W3M3:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/gcp-locations.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-3/gcp-locations.png

W3M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/behavioural-isolation.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-3/behavioural-isolation.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/geographic-isolation.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-3/geographic-isolation.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/heightweight2.gif -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-3/heightweight.gif
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-3/PCA1-smaller-smaller.gif -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-3/pca1.gif

W4M3:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-4/tree.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-4/tree.png 

W4M4:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-4/collection-sites-tz.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-4/collection-sites-tz.png

W5 -- should be all sorted. 

W6M3:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/figure1_resized-768x489.png  -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-6/w6m3-1.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/Screenshot%20from%202022-09-01%2010-37-53.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-6/w6m3-2.png

W7M1 (despite the paths):

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/429502_1_En_3_Fig4_HTML(1).jpg -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/w7m1-1.jpg
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/Screenshot%20from%202022-10-17%2009-41-44.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/w7m1-2.jpg
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-6/Screenshot%20from%202022-10-24%2019-22-07.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/w7m1-3.jpg

W7M2:

- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/introgression-cartoon.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/introgression-cartoon.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/spatial-gene-flow-cartoon.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/spatial-gene-flow-cartoon.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/adaptive-gene-flow.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/adaptive-gene-flow.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/haplotype-frequencies.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/haplotype-frequencies.png 
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/detecting-sweeps.sketchpad.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/detecting-sweeps.png
- https://storage.googleapis.com/vo_agam_release/reference/training-images/workshop-7/detecting-shared-sweeps.sketchpad.png -> https://vobs-resources.cog.sanger.ac.uk/training/img/workshop-7/detecting-shared-sweeps.png

**Comment by jonbrenas** [comment]
It is possible to figure it out from the references but I think it might help to have a list of PRs addressing this in order:

<s>W1M1: No linked image</s>
<s>W1M2: #247 (merged) </s>
<s>W1M3: No linked image</s>
<s>W1M4: #249 (merged) </s>

W2M1: #250
<s>W2M2: No linked image</s>
<s>W2M3: No linked image</s>
<s>W2M4: #252 (merged) </s>

<s>W3M1: No linked image</s>
<s>W3M2: No linked image</s>
W3M3: #253
W3M4: #258

<s>W4M1: No linked image</s>
<s>W4M2: No linked image</s>
W4M3: #256
W4M4: #257 

<s>W5M1: #239 (merged) </s>
<s>W5M2: No linked image</s>
<s>W5M3: No linked image</s>
<s>W5M4: #244 (merged)</s>

<s>W6M1: No linked image</s>
<s>W6M2: No linked image</s>
W6M3: #231
<s>W6M4: No linked image</s>

W7M1: #232
W7M2: #240
<s>W7M3: No linked image</s>
<s>W7M4: No linked image</s>

<s>W8M1: No linked image</s>
<s>W8M2: No linked image</s>
<s>W8M3: No linked image</s>
<s>W8M4: No linked image</s>
